ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -144,6 +144,15 @@ export NUM_WORKERS=2
 export PROV_BM_MAC=$(getExtraVal PROV_BM_MAC '')
 EOF
 
+    # Increase the WORKER_DISK for e2e-metal-ipi-ovn-upgrade
+    if [ "${JOB_NAME}" = "e2e-metal-ipi-ovn-upgrade" ]
```

Comment:
Looks like Job name has the full name in it

```suggestion
    # FIXME: Workaround to increase the WORKER_DISK for e2e-metal-ipi-ovn-upgrade
    if [ "${JOB_NAME}" = "periodic-ci-openshift-release-master-nightly-4.17-upgrade-from-stable-4.16-e2e-metal-ipi-ovn-upgrade" ]
```